### ISSUE TITLE
python -m aspen doesn't work correctly on windows

### ISSUE THREAD

**Issue by techtonik** [gratipay/aspen-python#440]
**Story**: I want to just run `aspen` (or `python -m aspen` FWIW) to run my website, see the URL and open it in browser.

**Comment by techtonik** [comment]
**Testing Aspen 0.37**:
This version doesn't have `aspen` command installed, but exposes `python -m aspen` module. Running it shows `Greetings, program! Welcome to port 8080.` Port is there, but no clickable URL. Attempt to open http://localhost:8080/ leads to Internal Error:
```
pid-2348 thread-3320 (MainThread) Reading configuration from defaults, environment, and command line.
pid-2348 thread-3320 (MainThread)   changes_reload         False                          default
pid-2348 thread-3320 (MainThread)   charset_dynamic        UTF-8                          default
pid-2348 thread-3320 (MainThread)   charset_static         None                           default
pid-2348 thread-3320 (MainThread)   configuration_scripts  []                             default
pid-2348 thread-3320 (MainThread)   indices                [u'index.html', u'index.json', u'index', u'index.html.spt', u'index.json.spt', u'index.spt'] default
pid-2348 thread-3320 (MainThread)   list_directories       False                          default
pid-2348 thread-3320 (MainThread)   logging_threshold      0                              default
pid-2348 thread-3320 (MainThread)   media_type_default     text/plain                     default
pid-2348 thread-3320 (MainThread)   media_type_json        application/json               default
pid-2348 thread-3320 (MainThread)   project_root           None                           default
pid-2348 thread-3320 (MainThread)   renderer_default       stdlib_percent                 default
pid-2348 thread-3320 (MainThread)   show_tracebacks        False                          default
pid-2348 thread-3320 (MainThread)   www_root               None                           default
pid-2348 thread-3320 (MainThread) project_root not configured (no template bases, etc.).
pid-2348 thread-3320 (MainThread) Renderers (*ed are unavailable, CAPS is default):
pid-2348 thread-3320 (MainThread)   STDLIB_PERCENT
pid-2348 thread-3320 (MainThread)   json_dump
pid-2348 thread-3320 (MainThread)   stdlib_format
pid-2348 thread-3320 (MainThread)   jsonp_dump
pid-2348 thread-3320 (MainThread)   stdlib_template
pid-2348 thread-3320 (MainThread) Greetings, program! Welcome to port 8080.
pid-2348 thread-3320 (MainThread) Traceback (most recent call last):
pid-2348 thread-3320 (MainThread)   File "C:\gratipay\inside\.v\lib\site-packages\algorithm.py", line 288, in run
pid-2348 thread-3320 (MainThread)     new_state = function(**deps.as_kwargs)
pid-2348 thread-3320 (MainThread)   File "C:\gratipay\inside\.v\lib\site-packages\aspen\algorithms\website.py", line 95, in get_resource_for_request
pid-2348 thread-3320 (MainThread)     return {'resource': resources.get(website, dispatch_result.match)}
pid-2348 thread-3320 (MainThread)   File "C:\gratipay\inside\.v\lib\site-packages\aspen\resources\__init__.py", line 176, in get
pid-2348 thread-3320 (MainThread)     raise entry.exc[0]  # TODO Why [0] here, and not above?
pid-2348 thread-3320 (MainThread) LoadError: Traceback (most recent call last):
pid-2348 thread-3320 (MainThread)   File "C:\gratipay\inside\.v\lib\site-packages\aspen\resources\__init__.py", line 168, in get
pid-2348 thread-3320 (MainThread)     entry.resource = load(website, fspath, mtime)
pid-2348 thread-3320 (MainThread)   File "C:\gratipay\inside\.v\lib\site-packages\aspen\resources\__init__.py", line 115, in load
pid-2348 thread-3320 (MainThread)     with open(fspath, 'rb') as fh:
pid-2348 thread-3320 (MainThread) IOError: [Errno 2] No such file or directory: u'C:\\gratipay\\inside\\'
pid-2348 thread-3320 (MainThread)
pid-2348 thread-3320 (MainThread) Failed to import pygments. Tracebacks won't be highlighted.
pid-2348 thread-3320 (MainThread) 500 Internal Server Error            /                        .\.v\lib\site-packages\aspen\www\error.spt
127.0.0.1 - - [18/Jun/2015 09:04:45] "GET / HTTP/1.1" 500 557
```

**Comment by whit537** [comment]
> This version doesn't have `aspen` command installed, but exposes `python -m aspen` module.

We're not going to add back an `aspen` command. We intentionally removed it in https://github.com/gratipay/aspen-python/pull/324. Aspen is a library.

> Port is there, but no clickable URL.

PRs accepted! :)

> Attempt to open http://localhost:8080/ leads to Internal Error:

Hypothesis: this is a Windows-specific dispatcher failure. We need to support Windows, which means the starting-point for addressing this bug is to implement Windows CI: https://github.com/gratipay/aspen-python/issues/442.

**Comment by pjz** [comment]
Fixed in #482, I'm pretty sure.  Reopen if that's not the case.
